api/src/main/resources/liquibase-update-to-2.3.xml

```diff
@@ -81,4 +81,17 @@
 								 referencedTableName="concept"
 								 referencedColumnNames="concept_id" />
     </changeSet>
+    <changeSet id="20190815-Trunk-5650" author="gitacliff">
+		<preConditions onFail="MARK_RAN">
+			<not>
+				<columnExists tableName="order_group" columnName="parent_order_group"/>				
+			</not>
+		</preConditions>
+		<comment>Adding "parent_order_group" column to order_group table</comment>
+        <addColumn tableName="order_group">
+            <column name="parent_order_group" type="int">
+            	<constraints nullable="true"/>
+            </column>
+        </addColumn>
```

Comment:
Shouldn't you add a foreign key constraint?

Reply:
oh true,,let me add it